package/tests/testthat/_snaps/stable/tables-ADA-adat04b.md

```diff
@@ -6,21 +6,21 @@
-        Baseline evaluable patients for ADA                  134           132             0           0           266    
-        Patients with a positive ADA sample at baseline      63             64             0           0           127    
-      Patients with a positive NAb sample at baseline         0             0              0           0            0     
-      Patients with no positive NAb sample at baseline        0             0              0           0            0     
-      Incidence of NAbs                                                                                                   
-        Post-baseline evaluable patients for ADA             134           132             0           0           266    
-        Patients positive for ADA                            66             59             0           0           125    
-      Patients positive for NAb                               0             0              0           0            0     
-      Patients negative for NAb                              134           132             0           0           266    
-      ————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+      —————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+                                                           A: Drug X   C: Combination   <Missing>   B: Placebo   All Drug X
+                                                            (N=134)       (N=132)         (N=0)      (N=134)      (N=266)  
+      —————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+      Baseline Prevalence of NAbs                                                                                          
+        Baseline evaluable patients for ADA                   134           132             0           0           266    
+        Patients with a positive ADA sample at baseline       63             64             0           0           127    
+        Patients with a positive NAb sample at baseline        0             0              0           0            0     
+        Patients with no positive NAb sample at baseline       0             0              0           0            0     
+      Incidence of NAbs                                                                                                    
+        Post-baseline evaluable patients for ADA              134           132             0           0           266    
+        Patients positive for ADA                             66             59             0           0           125    
+        Patients positive for NAb                              0             0              0           0            0     
+        Patients negative for NAb                             134           132             0           0           266    
```

Comment:
I think this was a bug (the previous indentation). cbind was losing the correct indentation

Reply:
Now, this is the correct indentation!! Wonderful